src/include/duckdb/function/scalar/string_functions.hpp

```diff
@@ -375,6 +375,16 @@ struct RegexpEscapeFun {
 	static ScalarFunction GetFunction();
 };
 
+struct PathJoinFun {
+	static constexpr const char *Name = "path_join";
+	static constexpr const char *Parameters = "path,...";
+	static constexpr const char *Description = "Joins path components using the platform separator, normalizing duplicate separators and dot segments. An absolute RHS overrides previous components.";
```

Comment:
I think this needs update -- absolute RHS now only overrides if a child, and no mention of strict mode in description, params, examples, etc.